README.md

```diff
@@ -15,7 +15,7 @@ newer version.
 ## Build it
 
 ```sh
-clojure -X:dev:build
+clj -X:build :project-dir "\"$(pwd)\""
```

Comment:
I think `clojure` is the lowest common denominator here. I've had `clojure` installed on a system but not `clj`. But never the other way around

Reply:
Yeah good catch. they will both always be installed by `clj` wraps with rlwrap and errors if it is not present.